horace_core/algorithms/resolution_plot.m

```diff
@@ -116,38 +143,46 @@
     if ~isempty(par{1}), proj = par{1}; end
     if ~isempty(par{2}), iax = par{2}; end
 elseif numel(par)~=0
-    error('Check the number and type of optional arguments')
+    error('HORACE:resolution_plot:invalid_argument',...
+        'Check the number and type of optional arguments')
 end
 
-if ~(isempty(proj) || isa(proj,'projaxes'))
+if ~isempty(proj) && ~isa(proj,'projaxes')
     proj = projaxes(proj);
 end
 
 if ~(isnumeric(iax) && (numel(iax)==2 || numel(iax)==3) &&...
-        numel(unique(iax))==numel(iax) && all(iax>=1) && all(iax)<=4)
-    error('Check axes indicies')
+        numel(unique(iax))==numel(iax) && all(iax>=1) && all(iax<=4))
+    error('HORACE:resolution_plot:invalid_argument',...
+        'Check axes indicies')
 end
 
 % - Plot target
-if present.current && isempty(findobj(0,'Type','figure'))
-    error('No current figure exists - cannot overplot')
-end
+% if present.current && isempty(findobj(0,'Type','figure'))
+%     error('No current figure exists - cannot overplot')
+% end
 
-if present.over
-    plot_args = {'over'};
+newplot = false;
+if present.noplot
+    plot_args = {'noplot'};
 elseif present.current
     plot_args = {'current'};
 elseif present.name
     plot_args = {'name',key.name};
 else
     plot_args = {};
+    newplot = true;
 end
 
 
-
 % Construct sqw object
 % --------------------
 % Create a two-dimensional sqw object with one pixel
+psi = psi_deg * (pi/180);
```

Comment:
Do we have a constant or a function anywhere for this common operation?

Reply:
Not that I know of. it would be handy though.